package-lock.json

```diff
@@ -1760,9 +1832,9 @@
       }
     },
     "node_modules/@stoplight/types": {
-      "version": "13.12.0",
-      "resolved": "https://registry.npmjs.org/@stoplight/types/-/types-13.12.0.tgz",
-      "integrity": "sha512-lpyB8b5cNcg/CyQ9W7Q2BxPkHhBA8+9byviGT6uL2Ok/5iao0poCnFUH++Cd7VN1/RaXrv0LT9fyOg4fJIl0iA==",
+      "version": "14.0.0",
```

Comment:
This increases the major version. We need to either ensure nothing affects us from the breaking change in their changelog or rather downgrade to the previous major.

Cc @magicmatatjahu so you are aware

Reply:
My bad, let me downgrade.

The breaking changelog entry is: `http-spec: IHttpService['security'] is a 2D array now`. Not sure whether that's relevant here, `IHttpService` doesn't appear in the code.